source/making_payments/index.html.md.erb

```diff
@@ -5,120 +5,140 @@ weight: 70
-fraudster could not alter the payment ID and intercept other users’ payments.
+### Create a secure random ID
 
-+ create a secure random ID (such as a UUID) and include this as part of the
-``return_url``, using a different ``return_url`` for each payment. Since a
-securely generated UUID is not guessable, fraudsters will not be able to
-intercept users’ payments.
+You can create a secure random ID,such as a UUID, and include this as part of
+the `return_url`. You should use a different ``return_url`` for each payment.
+Securely generated UUIDs are not guessable, so malicious users will
+not be able to securely generated UUID is not guessable, fraudsters will not
+be able to alter the payment ID and intercept other users’ payments.
 
->Note: If you create an ID yourself, you’ll likely need to store this in a
-datastore mapped to the payment ID just after you create a payment.
+If you create an ID yourself, you will likely need to store this in a datastore mapped
+to the payment ID just after you create a payment.
 
 ## Accepting returning users
 
- A user directed to the return URL could have:
+ A user directed to the `return_url` could have:
 
  - paid successfully
- - not paid because their card was rejected or they clicked cancel
- - not paid because your service (via the API) or service manager (via the self-service dashboard) cancelled the payment in progress
+ - not paid because their card was rejected or they selected __Cancel__
+ - not paid because your service (via the API) or service manager (via the GOV.UK Pay admin tool) cancelled the payment in progress
  - not paid because of a technical error
 
 Your service should use the API to check the payment status when the user
-reaches the return URL, and provide an appropriate response based on the final
+reaches the `return_url`, and provide an appropriate response based on the final
 status of the payment attempt.
 
 ## When a user does not complete their payment journey
 
-The user may close their browser or lose internet connection in the middle of
-the payment flow on GOV.UK Pay. These users will not be redirected back to
-your service.
+Users may close their browser or lose internet connection in the middle of
+their payment journey on GOV.UK Pay. In this event, users will not be
+redirected back to your service.
 
-You can still check on the status of these payments by making a GET request
-using the Location Header or Self Link, the same way you would if they were
-redirected, but just after a set time (eg, an hour).
+You can still [check on the status of these payments](#making-payments), but
+after a set time (eg, an hour).
 
->Note: GOV.UK Pay will expire incomplete payments after 90 minutes, but you should expect an occasional success or failure if the user experienced problems right at the moment of the redirect.
+GOV.UK Pay will expire incomplete payments after 90 minutes, but you should
+expect occasional notifications of success or failure if users experienced
```

Comment:
I wasn't sure how to word this, based on the original.

* Do we need this information?
* Can anyone think of a better way of wording it, if we do?

Reply:
I’m not really sure what this is trying to say.

Reply:
I've deleted it